### ISSUE TITLE
Interchange front-back and left-right in accordance with the right hand rule

### ISSUE THREAD

**Issue by ma-sadeghi** [PMEAL/OpenPNM#1575]
# Right hand rule (Blender manual)
<img src="https://user-images.githubusercontent.com/14086031/87323157-67c52f00-c4fc-11ea-9ebc-d06fa9b0fa5b.png" width=30%>

# Graph Algorithms and Applications I (textbook)
<img src="https://user-images.githubusercontent.com/14086031/87327567-36e7f880-c502-11ea-90c5-7e14660756a3.png" width=80%>

**Comment by jgostick** [comment]
how does this compare to numpy indexing and to paraview?

**Comment by ma-sadeghi** [comment]
## Create `Cubic` network from `numpy` array
```python
import numpy as np
import openpnm as op

im = np.ones((4, 5, 6))
net = op.network.CubicTemplate(im)
op.io.VTK.save(net, filename="net")
```
## Visualize in Paraview
<img src="https://user-images.githubusercontent.com/14086031/87327047-8ed22f80-c501-11ea-9cd4-8b6998841fbe.png" width=70%>

## Interchange left-right w/ front-back
```python
import numpy as np
import openpnm as op

im = np.ones((4, 5, 6))
net = op.network.CubicTemplate(im)

net["pore.left"], net["pore.front"] = net["pore.front"], net["pore.left"]
net["pore.right"], net["pore.back"] = net["pore.back"], net["pore.right"]

op.io.VTK.save(net, filename="net")
```
## Visualize in Paraview
<img src="https://user-images.githubusercontent.com/14086031/87328000-d6a58680-c502-11ea-931a-792a908fc375.png" width=70%>

**Comment by Ni2M** [comment]
OpenPNM: 
YZ front,back  deltaX
XZ left, right  deltaY
XY top,bottom  deltaZ
It should be:
YZ left, right  deltaX
XZ front, back deltaY
XY top, bottom deltaZ
working on a PR.

**Comment by Ni2M** [comment]
It is fixed in the PR #1660
